utils/pvacviz/app.py

```diff
@@ -54,10 +55,13 @@ def main():
         pass
 
     finally:
-        print(time.asctime(), "pVACviz server started at http://%s:%s" % (HOSTNAME, PORT))
+        print(time.asctime(), "pVACviz server started at {}".format(url))
```

Comment:
Is this really true in the `OSError` case? (Do we even want to try launching a browser in that case?)

Reply:
Good point. I just pushed a commit to fix this.